SR2023/2023-04-12-competition-wrapup.md

```diff
@@ -0,0 +1,64 @@
+| Challenges            | [Queen Mary's College](https://www.qmc.ac.uk/)
+| Online Presence       | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+
+## Event Media
+
+If you'd like some mementos of the competition or would just like to see the photos which were taken, we've collected them all together into [this photo album]().
+
+The livestreams of the matches are also available on YouTube, including footage from the final:
+
+- Day 1: https://www.youtube.com/watch?v=TxbfE9es-EY
+- Day 2: https://www.youtube.com/watch?v=Z3Y-bUSGw34
+
+The in-venue soundtrack playlists can also be found on [the event page](https://studentrobotics.org/events/sr2023/competition/#soundtrack).
+
+## Feedback
+
+We'd love to get feedback on how the competition was for your competitors this year, so we know what we did well, and where we can improve. Please share the below short, anonymous survey with your competitors.
+
+[Send us feedback](https://docs.google.com/forms/d/e/1FAIpQLScHuBLMCfTo9Yse8mYQMxCZxk6DSf5fa3aH98aGc4hbt980EA/viewform)
+
+Additionally, we welcome feedback from you the supervisors too. Please fill in this survey.
+
+[Supervisor Feedback](https://docs.google.com/forms/d/e/1FAIpQLSeiYyuAuEurcpENqskL78D84a6R-jH3N8D5Q_2nTYfYE_QtIQ/viewform)
+
+Without all of your invaluable feedback we wouldn't be able to run the competition and attract new sponsors.
+
+## Outstanding kit
+
+Whilst most of you will have returned your kit at the end of the competition event, you may still have the other pieces of Student Robotics kit around your college. If you were unable to attend the competition, this will likely be a full kit.
+
+If you do still have some of our kit, please let us know as soon as possible so we can organise its safe return. It is vitally important that all kit is returned to us so we can clean, test and re-issue it for future competitions.
+
+We will be contacting you soon if our records indicate you still have some items of kit.
+
+## Future Events
+
+We hope to run a competition next year and see you all there. We plan to open places around September and will let you know as soon as they are.
+
+In the meantime, be sure to follow us on Twitter [@StudentRobotics](https://twitter.com/studentrobotics) and [Instagram](https://www.instagram.com/student_robotics). If you haven't done so already, please sign up to [our general mailing list](https://studentrobotics.org/compete/), to be notified as soon as places are open.
+
+### Volunteering
+
+Student Robotics is powered by a collection of passionate individuals with varying skills from diverse backgrounds, many of whom are former competitors. We're all volunteers who give our spare time to make these events possible. If you have students graduating this year who may be interested in volunteering, please ask them to sign up to be a blueshirt at [srobo.org/join](https://srobo.org/join).
```

Comment:
Teachers may be interested in volunteering too, let's not exclude them :)

Reply:
This is true, although gets complicated if they want to be an active competition-side volunteer _and_ still lead a team.

Reply:
We already have multiple team leaders who are also volunteers. I don't think we should exclude them. We have also had supporting staff become volunteers, and the current wording here excludes anyone who isn't an 18 year old student...

Reply:
I agree we should word it more inclusively, two of our most crucial volunteers from SR2023 fall into this category.
I'm not sure how we'd word it, but we could direct those ones to /volunteer instead?